app/frontend/src/components/AcronymTooltip.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-tooltip right>
+    <template #activator="{ on }">
+      <v-icon v-on="on">help_outline</v-icon>
+    </template>
+    The Application Acronym must comply with the following format:
+    <ul>
+      <li>UPPERCASE LETTERS ONLY</li>
+      <li>Underscores may be placed between letters</li>
+      <li>Must begin and end with a letter</li>
+      <li>At least {{ fieldValidations.ACRONYM_MIN_LENGTH }} characters</li>
```

Comment:
The acronym filed in the database has a max length of 16 characters. I'm not sure why (webade related?)
I wonder if we should include that in the validation or instructions

Reply:
Yeah would have been webade... we've never listed the max length it looks like in those instructions. I'll adjust to reflect DB stuff 👍 

Probably can widen the field fine one day if ever needed but keeping short is good too IMO

Reply:
Fixed